app/scripts/controllers/edit/buildConfig.js

```diff
@@ -372,6 +377,44 @@ angular.module('openshiftConsole')
             };
           }
         );
+
+        DataService.list("configmaps", context, null, { errorNotification: false }).then(function(configMapData) {
+          configMapDataOrdered = orderByDisplayName(configMapData.by("metadata.name"));
+          $scope.valueFromObjects = configMapDataOrdered.concat(secretDataOrdered);
+        }, function(e) {
+          if (e.code === 403) {
+            return;
+          }
+
+          NotificationsService.addNotification({
+            id: "create-builder-list-config-maps-error",
+            type: "error",
+            message: "Could not load config maps.",
+            details: getErrorDetails(e)
+          });
+        });
+
+        DataService.list("secrets", context, null, { errorNotification: false }).then(function(secretData) {
+          secretDataOrdered = orderByDisplayName(secretData.by("metadata.name"));
+          $scope.valueFromObjects = secretDataOrdered.concat(configMapDataOrdered);
```

Comment:
```js
$scope.valueFromObjects = configMapDataOrdered.concat(secretDataOrdered);
```

Reply:
@spadgett, I gather https://github.com/openshift/origin-web-console/blob/master/app/scripts/controllers/create/createFromImage.js#L169 should be updated, then, too?

Reply:
Yes, looks like it.